app/(dashboard)/mailboxes/[mailbox_slug]/settings/team/deleteMemberDialog.tsx

```
@@ -0,0 +1,243 @@
+          variant: "destructive",
+          title: "Please select a valid assignee",
+        });
+        setLoading(false);
+        return;
+      }
+      try {
+        const handleRemoveMember = async () => {
+          try {
+            await removeTeamMemberAsync({ id: assignedToId.id, mailboxSlug });
+            return true;
+          } catch (error) {
```

ESLint

[error] 'error' is defined but never used. Allowed unused caught errors must match /^_/u.

(@typescript-eslint/no-unused-vars)